docs/kf_uids/uid_proposal.md

```diff
@@ -0,0 +1,38 @@
+# Proposal
+
+Based on [uid_considerations.md](uid_considerations.md) and [uid_migration.md](uid_migration.md)
+
+## 64 bit keys
+
+### Nodeless (much better than what we have now)
```

Comment:
From today, I think Nodeless is the better option and operationally easier to manage.